homeassistant/components/persistent_notification.py

```diff
@@ -57,6 +67,18 @@ def async_create(hass, message, title=None, notification_id=None):
     hass.async_add_job(hass.services.async_call(DOMAIN, SERVICE_CREATE, data))
 
 
+@callback
+def async_dismiss(hass, notification_id):
+    """Remove a notification."""
+    data = {
```

Comment:
There is only the notification ID.

```python
data = {ATTR_NOTIFICATION_ID: notification_id}
```
I guess that this was the point in this [comment](https://github.com/home-assistant/home-assistant/pull/7727/files/21a662191fbab89ee82e3bb0a01b03fad3463689#r118048834).